src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 
+@Component
 public interface FileReaderService {
```

Comment:
We need to annotate interface implementations, not the interface.

Reply:
done